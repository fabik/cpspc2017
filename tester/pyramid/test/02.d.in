2 50
5849794 1984769
233925723721
49035974263
170035577193
159129389811
11513989924
57987534970
593424787887
526471975080
523646014207
442066718418
465023549063
164326638024
325693627269
437015548514
174653545632
396189159856
642176904264
368985092804
615642969524
650428661590
170398245510
374155961099
522832409564
608878412062
464562401622
240912796570
460627962067
292725309729
27817246069
193047143706
640437226958
393624066057
404625112147
67998373021
132139844582
125276812427
244388657384
173487186836
79341493723
114657218526
395692753693
438108836651
521098653105
415006184538
174940145276
602148829488
383452134872
571220793852
388114676049
383971113428
