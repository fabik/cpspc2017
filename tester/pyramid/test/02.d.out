346668036620132618
72669241547343114
251985539526797808
235823030734019223
17063246475041290
85935076214161895
879430456924626353
780209217837101239
776021263251073320
655124194515161208
689145247352437625
243525132942886490
482664191475566313
647638574184690808
258829173593450303
587135591557917917
951679033082895016
546820313858227466
912356863147326024
963907788711367064
252522998645288014
554483322145347422
774815535554157210
902332074866100091
688461846333466094
357022583302782992
682631173186582220
433806538155319512
41223983090230750
286087708606821603
949100904835100798
583334230950546689
599637316340824577
100770715137893425
195825665308340382
185654941694221068
362173662134366914
257100679148820418
117580740647656119
169917152326129570
586399938603212953
649258780999813864
772246181751382926
615021626910475118
259253902153582622
892359117890498190
568259858809415610
846525076128049285
575169547790558357
569028963096572416
